README.md

```diff
@@ -1 +1,129 @@
+- [🤝 Contributing](#contributing)s
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 math-magician <a name="about-project"></a>
+
+**Math Magician Project** is a simple Project built through react npx create-react-app. User can interact by performing the calculator functions. In future versions there is going to complete website version of math magician Project will be developed using react components. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML CSS JavaScript React and redux
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[html css with JavaScript and React ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"> </a>
+
+- [Live Demo](Will be available after final approval)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+Web Browser, Code Editor.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd Math-Magician-Setup
+  git clone git@github.com:Kel-nana/Math-Magician-Setup.git
+```
+
+### Install
+VS CODE, Dev Dependencies (Linters), npm Package Manager, Webpack and Jest (for unit testing)
+
+### Usage
+Able to perform calculator operations and navigate through available links 
+
+### Run tests
+Able to perform calculator operations and navigate through available links 
+
+### Deployment
+The Project has been deployed in the link below.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Kelvin Irungu**
+
+- GitHub: [@Kel-nana](https://github.com/Kel-nana)
+- Linkedin: [@kelvin-irungu](https://www.linkedin.com/in/kelvin-irungu-838923249/)
+- Twitter: [@kelvin-irungu](https://twitter.com/home)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Functionality of calculator operations will be implemented using components. Going forward complete UI website will be developed with three links to navigate i.e Home Calculator and Quote.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please consider leaving a review.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partner [Abdul Ali and Zike]. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Your MIT license link currently leads to an error page. I have attached a screenshot below. I think this might be because you don't have a license file in your project. So I will suggest that you include an MIT license file in your project. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 

![image](https://user-images.githubusercontent.com/102699958/229872945-690eee38-7bd2-41d0-abc2-49265b2d0a52.png)

Reply:
implemented